templates/web/base/admin/user-form.html

```diff
@@ -33,6 +33,12 @@
         <input type="hidden" name="phone_verified" value="0">
       [% END %]
 
+      [% IF username_in_abuse %]
+      <li>
+            <p class="error">User is in Abuse table <input name="unban" type="submit" value="Unban"></p>
```

Comment:
Needs translation of both strings here (and if you used "User in abuse table" and tweaked that string to not have the brackets, it could share that translation).